extensions/adobe/experience/mobile-lifecycle-details.schema.json

```diff
@@ -0,0 +1,237 @@
+            }
+          }
+        }
+      }
+    },
+    "environment-ipV4": {
+      "properties": {
+        "xdm:environment": {
+          "type": "object",
+          "properties": {
+            "xdm:ipV4": {
+              "title": "IPv4",
+              "$ref": "https://ns.adobe.com/xdm/context/environment#/definitions/ipV4",
+              "description": "The numerical label assigned to a device participating in a computer network that uses the Internet Protocol for communication. "
+            }
+          }
+        }
+      }
+    },
+    "environment-ipV6": {
+      "properties": {
+        "xdm:environment": {
+          "type": "object",
+          "properties": {
+            "xdm:ipV6": {
+              "title": "IPv6",
+              "$ref": "https://ns.adobe.com/xdm/context/environment#/definitions/ipV6",
+              "description": "The numerical label assigned to a device participating in a computer network that uses the Internet Protocol for communication. "
+            }
+          }
+        }
+      }
+    },
+    "environment-domain": {
+      "properties": {
+        "xdm:environment": {
+          "type": "object",
+          "properties": {
+            "xdm:domain": {
+              "title": "Domain",
+              "$ref": "https://ns.adobe.com/xdm/context/environment#/definitions/domain",
+              "description": "The domain of the users ISP."
+            }
+          }
+        }
+      }
+    },
+    "environment-ISP": {
+      "properties": {
+        "xdm:environment": {
+          "type": "object",
+          "properties": {
+            "xdm:ISP": {
+              "title": "Internet service provider",
+              "$ref": "https://ns.adobe.com/xdm/context/environment#/definitions/ISP",
+              "description": "The name of the user's internet service provider."
+            }
+          }
+        }
+      }
+    },
+    "environment-language": {
+      "properties": {
+        "xdm:environment": {
+          "type": "object",
+          "properties": {
+            "dc:language": {
+              "title": "Language",
+              "$ref": "https://ns.adobe.com/xdm/context/environment#/definitions/dc:language",
+              "description": "The language of the environment to represent the user's linguistic, geographical, or cultural preferences for data presentation.\nLanguages are specified in language code as defined in [IETF RFC 3066](https://www.ietf.org/rfc/rfc3066.txt), which is part of BCP 47, which is used elsewhere in XDM."
+            }
+          }
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "#/definitions/mobile-lifecycle-details"
+    }
```

Comment:
In order to have all the properties defined available on UI to compose schema, you will have to add all the definitions in this allOf section. The way you added "$ref": "#/definitions/mobile-lifecycle-details". This definition will only expose the two properties defined under this.